torch/csrc/jit/passes/quantization.cpp

```diff
@@ -104,10 +107,29 @@ void addQuantDeQuantNodesForInput(Value* v, Node* n) {
   // Attach inputs to quant and dequant nodes
   quant->addInput(v);
   // Default Quant Params <Scale:1.0, ZeroPoint:0>
-  insertQuantNodeParams(quant, std::make_tuple(1.0, 0));
+  insertQuantNodeParams(quant, std::get<1>(qparam),
+    std::get<2>(qparam));
   dequant->addInput(quant->output());
 }
 
+template <typename... ArgT>
```

Comment:
Why do we need to templatize it when we know exactly what types we expect and they are always the same?

Reply:
no strong reason, its just this api is matching against key only, if in future tuple changes, this api can remain same. if it is major concern i can remove also